### PR TITLE
Support for cURL script generation

### PR DESCRIPTION
**Pull request by apoloa**
Added the class cURLBuilder that creates a cURL script to provide an example of each operation (endpoint).

### CODE REVIEW

File: lib/bundler.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ function getContentFromURL (url) {
 function getFileContent (filePath) {
   return new Promise((resolve, reject) => {
     fs.readFile(path.resolve(__dirname, filePath), (err, content) => {
-      if (err) {
+        if (err) {
```
Comment (fmvilas) [inline lib/bundler.js]: 👎

**Comment by apoloa** [comment]
@fmvilas Could you merge this?

**Comment by apoloa** [comment]
> LGTM. Just a minor comment. I'm happy to see you keep using it 😄

Of course, Why not? 🤣

**Comment by fmvilas** [comment]
@apoloa It's already on npm, version 0.10.0. Thanks for the PR!

**Comment by apoloa** [comment]
@fmvilas Thanks 💯

**Pull request merged**